### DISCUSSION TITLE
Onebot怎么连接早柚核心

### DISCUSSION THREAD

**Discussion by STESmly** [nonebot/discussions#478] [问答]
我现在的情况是，早柚核心部署了，刚开始用nb2的时候用的是v11的适配器，现在把v12的适配器也下好了，运行nb2时，cmd还是[SUCCESS] nonebot | OneBot V11 QQ号 | [message.group.normal]，但不知道怎么连接早柚核心（网页控制台能登陆了，但早柚核心的cmd没有群消息的接收）

**Comment by STESmly** [comment]
刚学着写nb2插件1个月，纯编程小白，希望能以一种简单易懂的方式交流，谢谢

**Comment by yanyongyu** [comment]
不太了解你说的早柚核心是什么

**Reply by STESmly** [reply]
> 不太了解你说的早柚核心是什么

https://docs.sayu-bot.com/是这个早柚核心

**Reply by yanyongyu** [reply]
看上去是bot项目，这个问题你应该问bot作者或者看bot自己的文档，这并不是nb相关的连接问题
